io.github.giantpinkrobots.bootqt.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.giantpinkrobots.bootqt
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+
+command: bootqt.sh
+
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
At least opening a file uses portals just fine, so I don't think this is necessary.

Reply:
Huh. I will have to try that once I get home.

Reply:
No it does not work without the knowledge of the exact path of the image file. I can make it filesystem=user folder only but then no other directory will be reachable.